frontend/packages/kubevirt-plugin/src/components/modals/vm-description-modal/vm-description-modal.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import { TextArea } from '@patternfly/react-core';
+
```

Comment:
unnecessary empty line

Reply:
there should be a line between node_modules and local imports

Reply:
Yes, this is a general Console convention - vendor (3rd party) imports should go first, then newline, then app specific imports. This helps to keep big import lists somewhat organized.